admin/acceptance.py

```diff
@@ -57,9 +61,41 @@ def remove_known_host(reactor, hostname):
     return run(reactor, ['ssh-keygen', '-R', hostname])
 
 
-def run_tests(reactor, nodes, control_node, agent_nodes, trial_args):
+def run_client_tests(reactor, node, trial_args):
+    """
+    Run the client acceptance tests.
+
+    :param INode node: The node to run client acceptance tests against.
+    :param list trial_args: Arguments to pass to trial. If not
+        provided, defaults to ``['flocker.acceptance']``.
+
+    :return int: The exit-code of trial.
+    """
+    if not trial_args:
+        trial_args = ['flocker.acceptance']
+
+    def check_result(f):
+        # TODO do we need to trap ProcessTerminated here?
+        # Test this when FLOC-1674 is merged
+        f.trap(ProcessTerminated)
+        if f.value.exitCode is not None:
+            return f.value.exitCode
+        else:
+            return f
+
+    return perform(dispatcher, run_remotely(
+        username=node.get_default_username(),
+        address=node.address,
+        commands=task_client_smoke_test()
+        )).addCallbacks(
+            callback=lambda _: 0,
```

Comment:
:question: It isn't consistent and I don't think that it is a "rule" but we have discussed that using `_` can be confusing and that it is better to use `ignored`.